Makefile

```diff
@@ -64,7 +64,9 @@ make-app-selinux:
 
 make-app: make-app-selinux
 
-make-all: make-app
+make-ipmi: make-ipmi-selinux
+
+make-all: make-app make-ipmi
```

Comment:
Yeah, I forgot to remove those - thanks for spotting it.
Fixed and squashed into 4b83bec182cbb82c28047c2b59657b81f894b183